### DISCUSSION TITLE
Azure AD users

### DISCUSSION THREAD

**Discussion by Lokeshmaxi** [provectus/kafka-ui#2244] [Q&A] [answered]
How to add azure AD users in the environment variable to access the Kafka-UI ,after integrating the SSO auth

**Comment by Haarolean** [comment, marked as answer]
Hey @Lokeshmaxi,

you can limit the access via dn pattern/userfilter.

https://github.com/provectus/kafka-ui/blob/master/documentation/compose/auth-ldap.yaml#L31
